src/WebUI/Resources.js

```diff
@@ -66,5 +61,14 @@ define("Environments/Providers/Kubernetes/Resources", ["require", "exports"], fu
     exports.WorkloadText = "Workload";
     exports.WorkloadDetails = "{0} details";
     exports.PodStatusPending = "Pending";
-    exports.MoreImagesText = "{0} and {1} more"
+    exports.MoreImagesText = "{0} and {1} more";
+    exports.LearnMoreKubeResourceText = "Learn more about Kubernetes resource";
+    exports.DeployKubeResourceText = "Use this resource to deploy workloads and services";

```

Comment:
DeployKubeResourceText  --> can we make this to come in one line? we can talk to PM around what is minimum width of the screen